source/class.cpp

```diff
@@ -1024,20 +1024,37 @@ std::string ClassBinder::bind_repr(Context &context)
 }
 
 
-string ClassBinder::bind_nested_classes(CXXRecordDecl const *EC, Context &context)
+void ClassBinder::for_public_nested_classes(const std::function<void(clang::CXXRecordDecl *C)>& f) const
```

Comment:
Looks like unction could take `const *` instead of just `*`. Also could you please use the same notation as the rest of the code, ie: `void ClassBinder::for_public_nested_classes(std::function<void(clang::CXXRecordDecl const *C)> const & f) const`? Thanks,